Fix the mapping from M3 platform to GNU configure target, for FreeBSD.
This particularly helps with m3gdb.

===================================================================
RCS file: /home/jdp/m3-cvs/m3/m3build/templates/PLATFORMS,v
retrieving revision 1.1.1.2
diff -u -r1.1.1.2 PLATFORMS
--- m3/m3build/templates/PLATFORMS.orig	1996/09/24 05:28:51	1.1.1.2
+++ m3/m3build/templates/PLATFORMS	1996/09/24 05:34:56
@@ -10,7 +10,7 @@
   "ARM"       : [ "POSIX", "32BITS", "arm--riscos"        ],
   "DS3100"    : [ "POSIX", "32BITS", "decstation"         ],
   "FreeBSD"   : [ "POSIX", "32BITS", "i486-unknown-bsd"   ],
-  "FreeBSD2"  : [ "POSIX", "32BITS", "i486-unknown-bsd"   ],
+  "FreeBSD2"  : [ "POSIX", "32BITS", "i486-unknown-freebsd" ],
   "HP300"     : [ "POSIX", "32BITS", "m68k-hp-hpux"       ],
   "HPPA"      : [ "POSIX", "32BITS", "hppa1.1-hp-hpux"    ],
   "IBMR2"     : [ "POSIX", "32BITS", "rs6000-ibm-aix3.2"  ],
